steps/step-copy-file/0.0.1/step.yml

```diff
@@ -0,0 +1,32 @@
+title: copy-file
+summary: |
+  Step to copy a file from one folder to another. You can use this for copying configuration files fromm archived location to a specific directory in the workspace.
+description: |
+  Step to copy a file from one folder to another. You can use this for copying configuration files from archived location to a specific directory in the workspace.
+website: https://github.com/satyasrikantmantha/bitrise-step-copy-file
+source_code_url: https://github.com/satyasrikantmantha/bitrise-step-copy-file
+support_url: https://github.com/satyasrikantmantha/bitrise-step-copy-file/issues
+published_at: 2017-08-29T00:31:45.079479601+05:30
+source:
+  git: https://github.com/satyasrikantmantha/bitrise-step-copy-file.git
+  commit: 11221535b026ba0fc31bb341e8146bd3c2dfabb1
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- utility
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- opts:
+    is_expand: false
```

Comment:
if `is_expand` is false, this input value will not be expanded (before your step would get it) . so if you would specify: `source_file_path : $HOME/path` in your step.sh source_file_path would hold: `$HOME/path` instead of `/users/user/path` (in your case this is not a breaking issue, since your are using this value in a bash script, so it will be resolved in your code, but in general: it is better in 99% to set `is_expand` to true)

Reply:
Hello @godrei,

Thank you very much for the review. I am a newbie with bitrise, and this is informative. Thank you also for explaining the is_expand property. I will update and it and submit it again. I will reshare.